larray/core/axis.py

```diff
@@ -614,6 +614,60 @@ def equals(self, other):
         return isinstance(other, Axis) and self.name == other.name and self.iswildcard == other.iswildcard and \
             (len(self) == len(other) if self.iswildcard else np.array_equal(self.labels, other.labels))
 
+    def min(self) -> Any:
+        """
+        Get minimum of labels.
+
+        Returns
+        -------
+        label
+            Label with minimum value.
+
+        Warnings
+        --------
+        Fails on non-numeric labels.
+
+        Examples
+        --------
+        >>> time = Axis('time=1991..2020')
+        >>> time.min()
+        1991
+
+        >>> country = Axis('country=Belgium,France,Germany')
```

Comment:
isn't having the Warning AND a failed example a bit overkill? Isn't the warning enough by itself?

Reply:
Depends on what users actually read. Some of them may only read the examples. 
It doesn't hurt to have both IMO.